cdap-ui/app/cdap/components/AbstractWidget/FormInputs/Select/index.tsx

```diff
@@ -49,6 +49,18 @@ const CustomizedInput = withStyles(() => {
   };
 })(InputBase);
 
+const CustomizedInput = withStyles(() => {
+  return {
+    input: {
+      padding: '7px 18px 7px 12px',
+      '&:focus': {
+        backgroundColor: 'transparent',
+        outline: `1px solid ${blue[100]}`,
```

Comment:
nit: I believe the theme object gets passed to `withStyles`? Like [here](https://github.com/cdapio/cdap/blob/develop/cdap-ui/app/cdap/components/AbstractWidget/FormInputs/Select/index.tsx#L29)

Reply:
Correct, fixed.